classes/Tools.php

```diff
@@ -3264,19 +3264,30 @@ public static function addonsRequest($request, $params = array())
 		return false;
 	}
 
-	public static function fileAttachment($input = 'fileUpload')
+	/**
+	 * Returns an array containing information about
+	 * HTTP file upload variable ($_FILES)
+	 *
+	 * @param string $input          File upload field name
+	 * @param bool   $return_content If true, returns uploaded file contents
+	 *
+	 * @return array|null
+	 */
+	public static function fileAttachment($input = 'fileUpload', $return_content = true)
```

Comment:
Hi, please set $return_content to false by default in order to keep original logic.

Best regards